interfaces/prompting/constraints.go

```diff
@@ -326,13 +330,13 @@ func (pm RulePermissionMap) validateForInterface(iface string, currTime time.Tim
 			invalidPerms = append(invalidPerms, perm)
 			continue
 		}
+		if err := entry.validate(); err != nil {
+			errs = append(errs, err)
```

Comment:
should we "continue" here?

Reply:
I think so too

Reply:
Good catch, thanks. I didn't fix the `continue`s when I reordered the validation/expiration checks. I'll confirm that the other place things were reordered is correct as well.

Reply:
Other one is correct. I've fixed this case, will push soon.